python/private/pypi/parse_requirements.bzl

```
@@ -32,6 +32,19 @@ load(":index_sources.bzl", "index_sources")
 load(":parse_requirements_txt.bzl", "parse_requirements_txt")
 load(":whl_target_platforms.bzl", "select_whls")
 
+def _extract_version(entry):
+    """Extract the version part from the requirement string.
+
+
+    entry: The requirement string.
+    """
+    version_start = entry.find("==")
+    if version_start != -1:
+        # Extract everything after '==' until the next space or end of the string
+        version = entry[version_start + 2:].split(" ")[0]
```

nit: I prefer using `partition` in the code because it just creates three return values.
```suggestion
        version, _, _ = entry[version_start + 2:].partition(" ")
```